hack/make/.build-deb/control

```diff
@@ -7,6 +7,7 @@ Vcs-Git: git://github.com/docker/docker.git
 Package: docker-engine
 Architecture: linux-any
 Depends: iptables, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Build-Depends: debhelper, dh-apparmor
```

Comment:
This line still isn't necessary -- https://github.com/docker/docker/pull/14609#discussion_r34596022
> our build-depends lives in the Dockerfiles directly, not in the package metadata